index.html

```diff
@@ -26,72 +32,83 @@
     <nav id="sidebar">
       <div>
         <div class="img logo rounded mb-30">
-          <a href="index.html" >
-            <p style="font-size:6.6rem;color:white;text-align:center;top:-53px;position:relative;">PACA</p>
-            <p style="display:inline-block;color:white;top:-128px;position:relative;text-align:justify;left:1em;font-size:24px"><span style="font-size:44px;margin-bottom:-37px;">PAN AFRICAN</span><span
-                style="margin-bottom:-33px;font-size:44px;word-spacing:86px;">COVID 19</span><span style="letter-spacing:11px;font-size:44px">ANALYSIS</span></p>
+          <a href="index.html">
+            <p style="font-size:5.6rem;color:white;text-align:center;top:-44px;position:relative;">PACA</p>
+            <p style="color:white;top:-6em;left:3em;position:relative;font-size:16px;">
+              Pan African COVID 19 Analysis
+            </p>
           </a>
         </div>
-        <div class="p-4 pt-5" style="max-height: 66vh;overflow: auto;">
+        <div style="background:white;">
+          <div class="logo1">
+            <img style="width:56%;" class="gg_logo" src="images/GG_Logo2.png">
+          </div>
+          <div class="logo2">
+            <img style="width:56%;" class="irish_logo" src="images/irish.png">
+          </div>
+        </div>
+        <div class="p-4 pt-5" style="overflow:auto;top:7em;position:relative;background:#606060;width:101%;">
           <div>
             <ul class="list-unstyled components mb-5">
               <li id="visualize" style="display:block">
-                <a href="#homeSubmenu" data-toggle="collapse" aria-expanded="false" class="dropdown-toggle">Infrastructure</a>
-                <ul class="collapse list-unstyled" id="homeSubmenu">
+                <a href="#homeSubmenu5" data-toggle="collapse" aria-expanded="false" class="highlight dropdown-toggle" onclick="hide()">COVID 19 cases</a>
+                <ul class="collapse list-unstyled" id="homeSubmenu5">
                   <li>
-                    <a href="#" onclick="ventilators_layer()">Ventilators</a>
+                    <a id="sec" href="#" onclick="cases_layer(); hide(); makeGreen();">Cases</a>
```

Comment:
@lynnnakiganda , please use another implementation of what you're trying to achieve here. ids in html should not be used like this.

Consider going through this article for clarity -> https://www.htmldog.com/guides/css/intermediate/classid/